Lua documentation for the skiller is broken

All targets that include lua doc fail. As an example, the `apidoc` target fails when building skill/lua documentation: 
```
$ make apidoc
[DOC] Building documentation (api.doxygen). This may take a while...
--> No warnings. Nice job.
[DOC] Generating luadoc for the skiller...
/usr/bin/lua: /usr/share/lua/5.3/luadoc/taglet/standard.lua:469: attempt to call a nil value (field 'foreachi')
stack traceback:
	/usr/share/lua/5.3/luadoc/taglet/standard.lua:469: in function 'luadoc.taglet.standard.start'
	/usr/share/lua/5.3/luadoc/init.lua:44: in function 'luadoc.main'
	(...tail calls...)
	/usr/bin/luadoc:121: in main chunk
	[C]: in ?
make: *** [/home/thofmann/code/fawkes/etc/buildsys/root/docs.mk:47: fawkes.luadoc] Error 1
```

--- a/etc/buildsys/config.mk
+++ b/etc/buildsys/config.mk
@@ -141,7 +141,7 @@ PKGCONFIG = $(shell which pkg-config)
 NM=nm
 ASCIIDOC=asciidoc
 ASCIIDOC_A2X=a2x
-LUADOC=luadoc
+LUADOC=ldoc
 ifneq ($(wildcard /bin/bash),)
   SHELL = /bin/bash
 else

